clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/model/ServiceCenterConfiguration.java

```diff
@@ -26,12 +26,23 @@ public class ServiceCenterConfiguration {
    */
   private Boolean ignoreSwaggerDifferent;
 
+  private Boolean rewriteSwagger;
+
   public Boolean isIgnoreSwaggerDifferent() {
     return ignoreSwaggerDifferent;
   }
 
+  public Boolean canRewriteSwagger(){
+    return rewriteSwagger;
+  }
+
```

Comment:
generate getter/setter, do not write yourself

Reply:
The return value will be used later